backend/src/handlers/actions.go

```diff
@@ -60,19 +68,25 @@ func (srv *Server) HandleImportUsers(w http.ResponseWriter, r *http.Request) {
 }
 
 func (srv *Server) HandleImportPrograms(w http.ResponseWriter, r *http.Request) {
+	logFields := log.Fields{
```

Comment:
I like this method of declaring the log fields, and making sure the handler name is one of them each time :+1: